spec/support/medical_copays/stub_medical_copays.rb

```diff
@@ -2,10 +2,21 @@
 
 def stub_medical_copays(method)
   let(:content) { File.read('spec/fixtures/medical_copays/index.json') }
-
   if method == :index
     before do
       allow_any_instance_of(MedicalCopays::VBS::Service).to receive(:get_copays).and_return(content)
     end
+  else
+    let!(:service) do
+      medical_copay_service = double
+      expect(MedicalCopays::VBS::Service).to receive(:new).and_return(medical_copay_service)
+      medical_copay_service
+    end
+    let(:statement_id) { '{93631483-E9F9-44AA-BB55-3552376400D8}' }
+    let(:content) { File.read('spec/fixtures/files/error_message.txt') }
+
+    before do
+      expect(service).to receive(:get_pdf_statement_by_id).with(statement_id).and_return(content)
```

Comment:
It seems like this should be a case statement, since it will only work for one endpoint.

Reply:
@kpethtel refactored to use case statement instead of if statements